### ISSUE TITLE
docwire does not seem to support languages other than English?

### ISSUE THREAD

**Issue by efieleke-tausight** [docwire/docwire#154]
I actually don't know if docwire supports multiple languages. If it does, the atached document trips it up.
[1718880709_ySg6a.odt](https://github.com/user-attachments/files/17494772/1718880709_ySg6a.odt)

**Comment by efieleke-tausight** [comment]
BTW, this is low priority as far as we are concerned.

**Comment by as-ascii** [comment]
Yes, it should work correctly. DocWire CLI can extract the text but we did only a simple quick test. What kind of problems was encountered? Some text is missing?

**Comment by as-ascii** [comment]
We double checked with the newest release with DocWire CLI utility and only minor layout differences are visible.

Text file exported from LibreOffice Writer:
[1718880709_ySg6a-libreoffice.txt](https://github.com/user-attachments/files/23343945/1718880709_ySg6a-libreoffice.txt)

Text file produced with DocWire CLI:
[1718880709_ySg6a-docwire.txt](https://github.com/user-attachments/files/23343949/1718880709_ySg6a-docwire.txt)

Diff file between them:
[1718880709_ySg6a-diff.txt](https://github.com/user-attachments/files/23343967/1718880709_ySg6a-diff.txt)
